common/proto-conv/tests/it/proto_conv.rs

```diff
@@ -114,6 +114,7 @@ fn new_table_info() -> mt::TableInfo {
             created_on: Utc.ymd(2014, 11, 28).and_hms(12, 0, 9),
             updated_on: Utc.ymd(2014, 11, 29).and_hms(12, 0, 10),
             comment: s("table_comment"),
+            field_comments: vec![],
```

Comment:
The test has to add some field comments and check if it encodes/decodes the table meta correctly.

Reply:
Ok.